src/fragments/lib/auth/ios/signout/20_global_signout.mdx

```diff
@@ -3,6 +3,8 @@
 <Block name="Async/Await">
 
 ```swift
+import AWSCognitoAuthPlugin
```

Comment:
Being extra picky, the function bellow will likely be defined inside a class/struct, and this import should be on the file level.